includes/admin/class-wc-payments-admin.php

```diff
@@ -54,7 +57,59 @@ public function add_payments_menu() {
 			return;
 		}
 
-		$top_level_link = $stripe_connected ? '/payments/deposits' : '/payments/connect';
+		$navigation_enabled = Loader::is_feature_enabled( 'navigation' );
+		$top_level_link     = $stripe_connected ? '/payments/deposits' : '/payments/connect';
+
+		if ( $navigation_enabled ) {
+			// Register category.
+			Menu::add_plugin_category(
+				[
+					'id'         => 'woocommerce-payments',
+					'title'      => __( 'WooCommerce Payments', 'woocommerce-payments' ),
+					'capability' => 'manage_woocommerce',
+				]
+			);
+
+			if ( ! $stripe_connected ) {
+				Menu::add_plugin_item(
+					[
+						'id'         => 'woocommerce-payments-connect',
+						'title'      => __( 'Connect', 'woocommerce-payments' ),
+						'capability' => 'manage_woocommerce',
+						'url'        => 'wc-admin&path=' . $top_level_link,
+						'parent'     => 'woocommerce-payments',
+					]
+				);
+			} else {
+				Menu::add_plugin_item(
+					[
+						'id'         => 'woocommerce-payments-deposits',
+						'title'      => __( 'Deposits', 'woocommerce-payments' ),
+						'capability' => 'manage_woocommerce',
+						'url'        => 'wc-admin&path=/payments/deposits',
+						'parent'     => 'woocommerce-payments',
+					]
+				);
+				Menu::add_plugin_item(
+					[
+						'id'         => 'woocommerce-payments-transactions',
+						'title'      => __( 'Transactions', 'woocommerce-payments' ),
+						'capability' => 'manage_woocommerce',
+						'url'        => 'wc-admin&path=/payments/transactions',
+						'parent'     => 'woocommerce-payments',
+					]
+				);
+				Menu::add_plugin_item(
+					[
+						'id'         => 'woocommerce-payments-disputes',
+						'title'      => __( 'Disputes', 'woocommerce-payments' ),
+						'capability' => 'manage_woocommerce',
+						'url'        => 'wc-admin&path=/payments/disputes',
+						'parent'     => 'woocommerce-payments',
+					]
+				);
```

Comment:
Currently, the last menu item is `Settings`, which links to a WP-Admin screen ([source](https://github.com/Automattic/woocommerce-payments/blob/master/includes/admin/class-wc-payments-admin.php#L100-L115)). Can that be done with the new navigation? We really need to keep that menu item, the full `WooCommerce -> Settings -> Payments -> WCPay` route is just too buried.

Reply:
Sure, just to confirm, you want the "Settings" item to appears directly under the "WooCommerce Payments" category, right?

Would this remove the item found under WooCommerce->Settings->Payments or both be kept?  Depending on the desired out come, we may need to do a little tinkering.

cc @elizaan36 @pmcpinto  for your thoughts on this as well.

Reply:
> Sure, just to confirm, you want the "Settings" item to appears directly under the "WooCommerce Payments" category, right?

We want it as the last item of the "WooCommerce Payments" category. So the WooCommerce Payments category will have 4 items: `Deposits`, `Transactions`, `Disputes`, and `Settings`. Only `Settings` will be a plain WP-Admin link, the rest of them will be WC-Admin fancy AJAX navigation links.

> Would this remove the item found under WooCommerce->Settings->Payments or both be kept? 

Both will be kept. Seasoned WooCommerce users will expect it to be in the `WooCommerce -> Settings -> Payments` list, but we also want a "shortcut" so new merchants don't have to discover that path by themselves. That's the current behavior and we'd very much want to keep it that way if possible :)

You can see [in the existing implementation](https://github.com/Automattic/woocommerce-payments/blob/master/includes/admin/class-wc-payments-admin.php#L100-L115) that we had to do some hacky stuff to get it working. I hope that's an option with the new navigation.

Reply:
> Both will be kept. Seasoned WooCommerce users will expect it to be in the WooCommerce -> Settings -> Payments list, but we also want a "shortcut" so new merchants don't have to discover that path by themselves. 

Having a shortcut to `WooCommerce -> Settings -> Payments -> WCPay` makes sense in the WCPay subnav, I agree that the payment gateway settings are hard to find otherwise.